### PR TITLE
For #11303 - Change copy login origin to open in browser

### PR DESCRIPTION
**Pull request by ekager**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/HomeActivity.kt [hunk 1]
```diff
@@ -475,7 +492,9 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         BrowserDirection.FromEditCustomSearchEngineFragment ->
             EditCustomSearchEngineFragmentDirections.actionGlobalBrowser(customTabSessionId)
         BrowserDirection.FromAddonDetailsFragment ->
-            EditCustomSearchEngineFragmentDirections.actionGlobalBrowser(customTabSessionId)
+            AddonDetailsFragmentDirections.actionGlobalBrowser(customTabSessionId)
```
Comment (ekager) [inline app/src/main/java/org/mozilla/fenix/HomeActivity.kt:495]: This looked like a bug so I fixed it 🤔

**Comment by ekager** [comment]
Old: 
Screen Shot 2020-07-07 at 1 57 35 PM

New: 
Screen Shot 2020-07-07 at 1 57 17 PM

**Comment by ekager** [comment]
The new open icon looks a little off so waiting for a new one from UX

**Comment by AmyYLee** [comment]
@ekager Here's the link to the new "Open in New" icon
https://drive.google.com/file/d/1YIMB5QXNmRiUV3wgob6mtdE1t18T8WrG/view?usp=sharing

**Comment by ekager** [comment]
new new :) 
Screen Shot 2020-07-09 at 5 37 45 PM

**Pull request merged**